index.html

```diff
@@ -0,0 +1,166 @@
+                    <li><a href="#"><img src="./assets/mobile/facebook.png" alt="fb-logo" width="10px" height="10px"></a></li>
+                    <li><a href="#"><img src="./assets/mobile/twitter.png " alt="twit-log" width="10px" height="10px"></a></li>
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Log Out</a></li>
+                </ul>
+        </div>
+        <nav>
+            <div class="container-main">
+                <div class="container-start second-nav">
+                    <a href="#"><label for="logo"><img class="logo" src="./assets/mobile/logo1.png" alt="logo"</label></a>
+                    <ul>
+                        <li><a class="active" href="#">Home</a></li>
+                        <li><a class="active" href="./about.html">About</a></li>
+                        <li><a class="active" href="#courses">Courses</a></li>
+                        <li><a class="active" href="#teachers">Teachers</a></li>
+                        <li><a class="active" href="#partners">Partners</a></li>
+                        <li class="bb-btn">
+                            <button class="cc-btn">Al-Noor Campaign</button>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+        <div class="mobile-option mobile-option2 ">
+            <div class="head-dom">
+              <button class="exit">
+                &#x2716;
+              </button>
+            </div>
+          <ul class="mob-style">
+            <li><a class="nav-link" href="#">Home</a></li>
+            <li><a class="nav-link" href="./about.html">About</a></li>
+            <li><a class="nav-link" href="#holyquran">Holy Quran</a></li>
+            <li><a class="nav-link" href="#courses">Courses</a></li>
+            <li><a class="nav-link" href="#teachers">Teachers</a></li>
+            <li><a class="nav-link" href="#partners">Partners</a></li>
+        </ul>
+    </header>
+    <main>
+        <section class="hero-section">
+            <div class="container-main">
+               <div class="container-start">
+                <div class="context">
+                    <img class="hamburger" src="./assets/mobile/hamburger.png" alt="menu">
+                    <p class="never">"Never forget who was there for you in difficult times"</p>
+                    <h1>Lets Learn Quran and Islamic Studies online</h1>
+                </div>
+                <div class="bg-color">
+                    <p>Al-Noor Academy is an accredited member school of the New England Association of Schools and Colleges (NEASC).</p>
+                </div>
+                <div class="date">
+                    <p class="big"><strong>2022.09.19(Monday) ~ 21(Thursday)</strong></p>
+                    <p class="small">@ AlNoor masjid Almaty, Qazaqstan<br> Akkent 56-66 </p>
+                </div>
+        </div> 
+        </section>
```

Comment:
- [ ] This section doesn't match the design. Please make sure you are following the design. 

Cloned
![image](https://user-images.githubusercontent.com/58504655/192089286-e2a1cefc-d6b9-4f47-8f3e-32c7461610e5.png)

Original
![image](https://user-images.githubusercontent.com/58504655/192089274-bfff26e3-0dfa-4ad3-88e7-b9036deaaba1.png)

Reply:
As I mentioned before following the design 100% is not a requirement! **Small adjustments like text or image changes are acceptable.**

Reply:
- [ ] @NurkaAmre great work on your project however as stated in the requirement, the page should look almost identical to the design given, the color, typography, and element layout. In your design the color doesn't match the given design color which I noted is the case on both pages, , below is a comparison, :

![image](https://user-images.githubusercontent.com/61408860/192106440-891fc832-2d45-47dc-9cdc-261d236e89b0.png)

Reply:
@danmainah u want i change only color?